Automattic/jetpack
# Carousel - fix bug with carousel swiper clashing with slideshow swiper

Fixes #20263

#### Changes proposed in this Pull Request:
* Give carousel swiper instance a unique class

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check our PR and add a standard gallery block and a slideshow block to a single post
* Make sure that both galleries work correctly in the frontend

Before:
![before-swiper](https://user-images.githubusercontent.com/3629020/124518177-e6b42580-de39-11eb-881c-af57c9a7dcff.gif)

After:
![after-swipe](https://user-images.githubusercontent.com/3629020/124518182-eae04300-de39-11eb-9fd0-651dbb4c2a1d.gif)
